target/snitch_cluster/Makefile

```
@@ -49,12 +49,15 @@ ifeq (${CFG_OVERRIDE}, cfg/snax-gemm.hjson)
 endif
 
 ifeq (${CFG_OVERRIDE}, cfg/snax-streamer-gemm.hjson)
-	SNAX_DEV_ROOT ?= $(shell $(BENDER) path snax-streamer-gemm-dev)
-	include $(SNAX_DEV_ROOT)/Makefile
+	SNAX_GEMM_ROOT ?= $(shell $(BENDER) path snax-gemm)
```

```suggestion
	SNAX_GEMM_ROOT ?= $(shell $(BENDER) path snax-streamer-gemm)
```
Is it possible to rename the bender target to `snax-streamer-gemm`? This way the name for the accelerator is fixed everywhere and will not cause confusion. Also then this Makefile script is easier to automate like in #112 .